hack/e2e.sh

```diff
@@ -27,6 +27,10 @@ mkdir -p $TEST_DIR
 
 source "${SCRIPT_DIR}/libe2e.sh"
 
+# Mandatory environment variables
+: ${CHART_VALUES:?}
+: ${CHART_VALUES_CASSANDRA:?}
```

Comment:
Given we can now toggle which e2e tests to run (i.e. I could not run cassandra e2e), this var isn't *always* required. I can see it's defaulted in the Makefile itself, but would it make sense to move some of these defaults into this file? (perhaps including CHART_VALUES)?

Reply:
Good idea. Done.